templates/section_page.hbs

```diff
@@ -19,6 +19,22 @@
         {{/if}}
       </header>
 
+      {{#if section.sections}}
+        <ul class="section-list section-list--collapsed">
+          {{#each section.sections}}
+          <li class="section-list-item">
+              <a href="{{url}}">
+                <span>{{name}}</span>
+                <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" focusable="false" viewBox="0 0 16 16" aria-hidden="true">
```

Comment:
can this be added as a separate file similar to https://github.com/zendesk/copenhagen_theme/blob/master/assets/dropdown-arrow.svg ?

Reply:
I'd lean in favour of inline `svg`s, this way, it will take in the color of the parent, in this case `text-color`. I'd like @luis-almeida's opinion on this.

Reply:
I tend to agree that cutting the extra network request, and most importantly being able to use `currentColor`, justifies inlining the `svg`.
Would be nice to do the switch for `dropdown-arrow.svg` as well for consistency and at some point have a build tool that can do this for us.